package.json

```diff
@@ -37,7 +37,7 @@
     "bluebird": "^2.9.12",
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
-    "loopback-datasource-juggler": "^3.0.0",
+    "loopback-datasource-juggler": "git+https://github.com/ebarault/loopback-datasource-juggler.git",
```

Comment:
Please revert. If this pull request depends on changes from @ebarault's fork, then those changes must be landed first, before this pull request can be accepted.